tmpl/procaddr.tmpl

```diff
@@ -3,7 +3,7 @@
 //
 // windows: WGL
 // darwin: CGL
-// linux: GLX
+// (the other POSIX): GLX
```

Comment:
Please list them explicitly, it's only 2 builds tags. It'll be more clear in the future.